converters/fromZigbee.js

```diff
@@ -5127,6 +5127,37 @@ const converters = {
             }
         },
     },
+    tuya_alecto_smoke: {
+        cluster: 'manuSpecificTuya',
+        type: ['commandDataResponse', 'commandDataReport'],
+        convert: (model, msg, publish, options, meta) => {
+            const dpValue = tuya.firstDpValue(msg, meta, 'tuya_alecto_smoke');
+            const dp = dpValue.dp;
+            const value = tuya.getDataValue(dpValue);
+            switch (dp) {
+            case tuya.dataPoints.alecto_Smoke_state:
+                return {"Smoke state": {0: "Alarm", 1: "Normal"}[value]};
```

Comment:
Please change "Smoke state" -> `smoke_state` and make the values lowercase (e.g. `Alarm` -> `alarm`)